### PR TITLE
Win dependencies - openvino

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA CVS-157903
Adding windows_install_dependencies.bat
It will install openvino and update workspace for the location in CI.

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: windows_install_dependencies.bat [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+:: Set proper PATH environment variable: Remove other python paths and add c:\opt with bazel, wget to PATH
+set "PATH=%setPath%"
+
+:: OpenVINO
+set "openvino_dir=w_openvino_toolkit_windows_2024.5.0.17288.7975fa5da0c_x86_64"
+set "openvino_ver=w_openvino_toolkit_windows_2024.5.0.17288.7975fa5da0c_x86_64.zip"
```
Comment (mzegla) [inline windows_install_dependencies.bat:28]: Can we update to 2025.0?

File: windows_install_dependencies.bat [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+set "openvino_http=https://storage.openvinotoolkit.org/repositories/openvino/packages/2024.5/windows/"
+set "openvino_wget=wget -P %BAZEL_SHORT_PATH%\ %openvino_http%%openvino_ver%"
+set "openvino_zip=%BAZEL_SHORT_PATH%\%openvino_ver%"
+set "openvino_workspace=C:\\\\opt\\\\intel\\\\openvino\\\\runtime"
+set "openvino_new_worksapce=C:\\%1\\openvino\\runtime"
```
Comment (mzegla) [inline windows_install_dependencies.bat:33]: why do we need this?

File: windows_install_dependencies.bat [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+:: Set proper PATH environment variable: Remove other python paths and add c:\opt with bazel, wget to PATH
+set "PATH=%setPath%"
+
+:: OpenVINO
+set "openvino_dir=w_openvino_toolkit_windows_2024.5.0.17288.7975fa5da0c_x86_64"
+set "openvino_ver=w_openvino_toolkit_windows_2024.5.0.17288.7975fa5da0c_x86_64.zip"
+set "openvino_http=https://storage.openvinotoolkit.org/repositories/openvino/packages/2024.5/windows/"
+set "openvino_wget=wget -P %BAZEL_SHORT_PATH%\ %openvino_http%%openvino_ver%"
```
Comment (mzegla) [inline windows_install_dependencies.bat:30]: Why do we keep this command in the variable, but other commands are used directly?

File: windows_install_dependencies.bat [hunk 4]
```diff
@@ -0,0 +1,56 @@
+::
```
Comment (mzegla) [inline windows_install_dependencies.bat:1]: Is this supposed to be executed in CI only or also by the user? If it's for the user as well, perhaps we should include it in the docs along with the information that `wget` and `tar` are needed for this to work.

File: windows_install_dependencies.bat (hunk 2 above)
Comment (rasapala) [reply windows_install_dependencies.bat:33]: We need to update path to openvino libs in bazel workspace file in the Ci.

File: windows_install_dependencies.bat (hunk 4 above)
Comment (rasapala) [reply windows_install_dependencies.bat:1]: I will update the documentation once the script installs all of the dependencies, currently lets stick to manual install instructions.

**Review by mzegla — APPROVED** [review]

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**